cache_checker.js

```diff
@@ -122,16 +122,21 @@ is_cached = async (URL, flag) => {
 
   //Print the Results
   console.log(dataHashMap);
+  console.log(
+    "Total Request Made:",
+    dataHashMap.count.cssCount +
+      dataHashMap.count.imageCount +
+      dataHashMap.count.fontCount +
+      dataHashMap.count.jsCount
+  );
   console.log(is_leverage_cache);
   browser.close();
-  console.log('Completed!');
+  console.log("Completed!");
 
   /* To check if the URL has SSL certificate */
-  const options = updatedURL;
-
+  const options = URL;
```

Comment:
do not change this, use `updatedURL` instead of `URL`

Otherwise, it throws exception while directly using  `github.com`